modules/navier_stokes/doc/content/source/physics/WCNSFVTwoPhaseMixturePhysics.md

```diff
@@ -0,0 +1,60 @@
+# Navier Stokes Two Phase Mixtures / WCNSFVTwoPhaseMixturePhysics
+
+!syntax description /Physics/NavierStokes/TwoPhaseMixture/WCNSFVTwoPhaseMixturePhysics
+
+## Equation(s)
+
+This [Physics](Physics/index.md) adds terms to the flow and energy equations to account for the presence of a
+two-phase mixture. If specified with the [!param](/Physics/NavierStokes/TwoPhaseMixture/WCNSFVTwoPhaseMixturePhysics/add_phase_transport_equation)
+parameter, it can also solve for the advection-diffusion equation of a moving phase fraction.
+
+!alert note
+If the other phase is solid, the [!param](/Physics/NavierStokes/TwoPhaseMixture/WCNSFVTwoPhaseMixturePhysics/add_phase_transport_equation)
+parameter should be set to false.
+
+The phase advection-diffusion equation is:
+
+!equation
+\dfrac{\partial \phi}{\partial t} + \nabla \cdot (\phi \mathbf{v}) - \nabla \cdot (k \nabla \phi) -\alpha \phi = 0
+
+where:
+
+- $\phi$ is the phase fraction
+- \mathbf{v} is the advecting velocity
+- $k$ the phase diffusivity
+- $\alpha$ is the phase exchange coefficient
+
+The kernels created are:
+
+- [FVFunctorTimeKernel.md] for the time derivative for a transient solve
+- [INSFVScalarFieldAdvection.md] for the scalar advection term
+- [FVDiffusion.md] for the scalar diffusion term
+
+The momentum equations, if defined using a [WCNSFVFlowPhysics.md], are modified in the presence of a two-phase
+mixture. Density and viscosity should be set to their mixture values, see [#materials] for more information.
```

Comment:
Also mention and link to the fluid energy physics since you discuss its modification below as well

Reply:
done